.github/workflows/coverage.yml

```
@@ -77,7 +77,7 @@ jobs:
           .github/actions/init_initialize-imap-fixtures.sh
           docker-compose exec -T app .github/actions/test_tests-imap.sh
       - name: "Codecov"
-        uses: "codecov/codecov-action@v3.1.4"
+        uses: "codecov/codecov-action@v3.1.5"
```

Will prevent creation of PRs for each minor version.
```suggestion
        uses: "codecov/codecov-action@v3"
```